metagpt/actions/action_node.py

```diff
@@ -86,7 +86,7 @@ class ReviseMode(Enum):
 {constraint}
 
 ## action
-generate output and make sure it follows the format example.
+Follow format example's json format, generate output and make sure it follows the format example.
```

Comment:
the word "json" or "markdown" depends on user's choice.

Reply:
add `prompt_schema` for user's choice